### ISSUE TITLE
Show time left instead

### ISSUE THREAD

**Issue by miesismore** [ajyang818/prioritab#40]
Hi, love this extension, just wondering if exist an easy way of showing the time left instead of time consumed,  i guess is related with editing some JS, would appreciate some help. Best regards.

**Comment by ajyang818** [comment]
Yep, this should be possible. It's been a while but I believe I used a JS library called moment.js to help with time calculations. That should give you all the tools you need to get 'time left'

**Comment by miesismore** [comment]
Will look into it, thank you!!

**Comment by miesismore** [comment]
I modified the maths in prioritab.min, worked for the moment, thank you :)
